docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -137,17 +152,58 @@ Relogin to fetch updated certificates:
 $ tsh logout; tsh login --proxy=example.com --user=alice
 ```
 
-Having the necessary permissions, let's add a device to the inventory. Replace
-the `$SERIAL` below with your macOS device serial number (visible under Apple ->
-"About This Mac" -> "Serial number", or use the command below).
+Before we can enroll the device, we need to register it. In order to register
```

Comment:
```suggestion
Before you can enroll the device, you need to register it. To register
```
I see that Teleport docs use a lot of first person plural (we). It's a little unusual except in tutorials. My suggestion is more "typical" but feel free to ignore if the point of view feels right to you.

Reply:
Definitely prefer "you" here, thanks for pointing this out!